src/invidious/helpers/extractors.cr

```diff
@@ -550,7 +596,10 @@ def extract_items(initial_data : Hash(String, JSON::Any), author_fallback : Stri
 
   # This is identical to the parser cycling of extract_item().
   ITEM_CONTAINER_EXTRACTOR.each do |extractor|
+    LOGGER.trace("extract_items: Attempting to extract item container using \"#{extractor.extractor_name}\" (cycling...)")
+
     if container = extractor.process(unpackaged_data)
+      LOGGER.debug("extract_items: Successfully unpacked container with \"#{extractor.extractor_name}\"")
       # Extract items in container
       container.each do |item|
         if parsed_result = extract_item(item, author_fallback, author_id_fallback)
```

Comment:
Ideally, we shouldn't prevent the entire page from loading when only a few of the `extract_item` failed. Instead, I think just having a simple banner like this on the top:
```
Looks like not all of the items have been parsed. Please copy the following logs to Invidious on Github!
<details box with the standard Invidious error>
```

With all of the successfully parsed content below would be sufficient.